Dockerfile

```diff
@@ -18,13 +16,15 @@ RUN apk add --no-cache \
 
 ARG HUGO_VERSION
 
-RUN mkdir $HOME/src && \
-    cd $HOME/src && \
+RUN mkdir /src && \
+    cd /src && \
     curl -L https://github.com/gohugoio/hugo/archive/refs/tags/v${HUGO_VERSION}.tar.gz | tar -xz && \
-    cd "hugo-${HUGO_VERSION}" && \
+    cd hugo-${HUGO_VERSION} && \
     go install --tags extended
 
-FROM golang:1.18-alpine
+FROM alpine:3.14
+
+LABEL maintainer="Luc Perkins <lperkins@linuxfoundation.org>"
```

Comment:
I'm not sure we need to call out an individual maintainer here.

Reply:
> I'm not sure we need to call out an individual maintainer here.

@nate-double-u You are correct that calling out an individual maintainer in the Dockerfile is not necessary. The maintainer label is optional, and if included, it should typically identify a group or organization responsible for maintaining the image rather than an individual.